src/routes/resources/resources.routes.ts

```
@@ -262,6 +263,65 @@ export const upvoteQueue = createRoute({
+      resourceId: z.string().min(3),
+    }),
+  },
+  responses: {
+    [HttpStatusCodes.CREATED]: jsonContent(
+      z.object({
+        success: z.boolean().default(true),
+        resourceId: z.string(),
+        message: z.string(),
+      }),
+      "The Bookmark added"
+    ),
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)